connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/operate/OperateClientAdapterImpl.java

```diff
@@ -98,7 +99,7 @@ public List<FlowNodeInstance> fetchActiveProcessInstanceKeyByDefinitionKeyAndEle
       return result.stream()
           .map(
               node ->
-                  new FlowNodeInstance(
+                  new io.camunda.connector.runtime.core.inbound.FlowNodeInstance(
```

Comment:
Can we use directly the new class [FlowNodeInstance](https://github.com/camunda-community-hub/spring-zeebe/blob/main/camunda-sdk-java/java-client-operate/src/main/java/io/camunda/operate/model/FlowNodeInstance.java) here? (so that we can just use 1 class)

It has the same name and same fields now. Thoughts?

Reply:
Yes, of course.